crypto/evp/p_rsa_asn1.c

```diff
@@ -152,6 +165,19 @@ static int rsa_priv_decode(EVP_PKEY *out, CBS *params, CBS *key) {
   return 1;
 }
 
+static int rsa_pss_priv_decode(EVP_PKEY *out, CBS *params, CBS *key) {
```

Comment:
Same comments as for the `public_decode` function

Reply:
This is also used by EVP_PKEY_RSA EVP_PKEY_ASN1_METHOD. If fips is an issue, separate story should be created.

Reply:
Was asking because I don't know. Something we should research more.